### ISSUE TITLE
TrainResNet_ImageNet_Distributed stopped after 1 epoch

### ISSUE THREAD

**Issue by height1987** [Microsoft/CNTK#3355]
My python script is [TrainResNet_ImageNet_Distributed.py](https://github.com/height1987/resnet_20_map/blob/master/TrainResNet_ImageNet_Distributed.py) 

the command is   `python TrainResNet_ImageNet_Distributed.py -n resnet34`
and I also tried `mpiexec -n 2 python TrainResNet_ImageNet_Distributed.py -n resnet34`

the process stopped after one epoch,and the log :

-------------------------------------------------------------------
Build info:

                Built time: Apr 13 2018 22:23:40
                Last modified date: Fri Apr 13 20:27:14 2018
                Build type: release
                Build target: CPU-only
                With ASGD: yes
                Math lib: mkl
                Build Branch: HEAD
                Build SHA1: 11c82c8019b78323b4f5169a2d8c90fba5ae49da
                MPI distribution: Open MPI
                MPI version: 1.10.7
-------------------------------------------------------------------
Selected CPU as the process wide default device.
Restoring training session from the checkpoint '/opt/cntk/Examples/Image/DataSets/Warehouse/my-2/Models/ResNet_ImageNet.model'
Start training: quantize_bit = 32, epochs = 90, distributed_after = 0
Finished Epoch[1 of 90]: [Training] loss = 0.682990 * 32, metric = 43.75% * 32 10.147s (  3.2 samples/s);
Finished Evaluation [1]: Minibatch[1-292]: metric = 46.45% * 9313;

I'm using my own images when training 
am I missing something?

any help would be appreciated.
